example/index.html

```diff
@@ -41,6 +43,13 @@
             <invenio-search search-endpoint="http://localhost:5000/api/records/">
               <div class="row">
                 <div class="col-md-3">
+                  <invenio-search-range
+                   template="/src/invenio-search-js/templates/range.html"
+                   histogramid="#year_hist"
```

Comment:
can we pass all options as `JSON`? i.e.

```html
<invenio-search-range
  template="/src/invenio-search-js/templates/range.html"
  options='{"histogramID": "#year_hist", "width": "180"}'
></invenio-search-range>
```
is pretty handy for passing options with a Jinja ``config`` variable